spec/requests/api/v1/authentication_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe 'Auth', type: :request do
+
+  describe 'POST /api/v1/auth/sign_in' do
+
+    let(:headers) { { HTTP_ACCEPT: 'application/json' } }
+    let!(:user) { FactoryGirl.create(:user) }
+    let(:credentials) { user.create_new_auth_token }
+
+    it 'valid credentials returns status 200' do
+      post '/api/v1/auth/sign_in', { email: user.email, password: user.password }, headers
```

Comment:
I would place this `post` request in a `before` block since you are using the same request in to `it` blocks.